lib/studies/crane/app.dart

```diff
@@ -19,6 +19,7 @@ class CraneApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      restorationScopeId: 'appNavigator',
```

Comment:
I know you mentioned this ID doesn't matter but for consistency this should be craneApp I guess